aem-cif-product-page-extension/extension/index.js

```diff
@@ -16,3 +16,5 @@ export { AfterProductData } from './src/components/AfterProductData';
 export { ProductTitle } from './src/components/ProductTitle';
 export { default as linkWrapper } from './src/linkWrapper';
 export { applyCustomization } from './customizer';
```

Comment:
This line is no longer valid, since you replaced `applyCustomization` with `applyProductPageCustomization` and `applyCategoryPageCustomization`.